system-addon/content-src/styles/_icons.scss

```diff
@@ -75,7 +75,7 @@
 
   &.icon-trending {
     background-image: url('#{$image-path}glyph-trending-16.svg');
-    transform: translateY(2px); /* trending bolt is visually top heavy */
+    transform: translateY(2px); // trending bolt is visually top heavy
```

Comment:
I suppose to be explicit, the `//` quotes are removed from the generated file whereas `/* */` comments remain. Just making sure that's desired. @k88hudson ?